docs/appendices/environment_vars.rst

```diff
@@ -91,8 +91,10 @@ There are several environment variables that affect the way Toil runs.
 |                                  | used with the Kubernetes batch system, if the      |
 |                                  | credentials allow access to S3 and SimpleDB.       |
 +----------------------------------+----------------------------------------------------+
-| TOIL_AWS_ZONE                    | The EC2 zone to provision nodes in if using        |
-|                                  | Toil's provisioner.                                |
+| TOIL_AWS_ZONE                    | Zone to use when using AWS. Also determines region.|
++----------------------------------+----------------------------------------------------+
+| TOIL_AWS_REGION, AWS_REGION,     |                                                    |
+| DEFAULT_AWS_REGION               | Region to use when using AWS.                      |
```

Comment:
Right now I'm having it check all three of these region variables in the order I give here, and using the first one that matches. If that doesn't turn anything up, I get the region from the zone.

I added the other variable name options because I think Fargate sets one of them by default, so if we consult it we can auto-magically pick up the current region in AGC and not have to worry about talking to the metadata service or setting TOIL_AWS_REGION in the AGC startup script. But if it's confusing I can go back to just `TOIL_AWS_REGION`.

I'm not sure what happens if you set a zone and a region that don't match, and I don't think any good can come of it. Right now that's Undefined Behavior. I'll see if I can avoid it: our last way to get the zone is to just pick "a" in Boto's configured region, so we could change that to just pick "a" in whatever region we are using.